deploy/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+## **deploy@3.1.0** (2024-10-04)
```

Comment:
@lorenjohnson I do not remember what the reasoning behind making this a minor bump. Technically, there are no new feature, are there?